lib/mssql.js

```diff
@@ -2,12 +2,12 @@ import Ajv from "ajv";
 import JSONStream from "JSONStream";
 import {json} from "micro";
 import mssql from "mssql";
-import {failedCheck, badRequest, notImplemented} from "./errors.js";
 import {Transform} from "stream";
 
-const TYPES = mssql.TYPES;
+import {failedCheck, badRequest, notImplemented} from "./errors.js";
+import pools from "./pools.js";
 
-const pools = new Map();
+const TYPES = mssql.TYPES;
```

Comment:
import mssql from "mssql";
```suggestion
import mssql, {TYPES} from "mssql";
```